coeus-impl/src/main/resources/ApplicationResources.properties

```diff
@@ -826,9 +827,9 @@ error.award.awardhierarchy.sync.invalidstatus=The Award {0} must have an active
 error.award.awardhierarchy.sync.locked=This Award has been locked for an Award Hierarchy Sync operation by document number {0}. 
 
 #START - Overriden rice properties
-app.feedback.linkText=Provide Feedback
+app.feedback.linkText=Get Help
 #To not show a feedback link, simply leave the feedback link blank.
-app.feedback.link=https://test.kuali.org/jira/secure/CreateIssue.jspa?pid=10310&issuetype=1
+app.feedback.link=mailto:support@rsmart.com?Subject=Help
```

Comment:
Can we make this a parameter instead so implementers can change it in the app? If not, we definitely shouldn't link out to an rsmart address.